src/main/java/com/eppo/sdk/EppoClient.java

```diff
@@ -171,33 +193,82 @@ public Optional<String> getAssignment(String subjectKey, String experimentKey,
      * @return
      */
     public Optional<String> getAssignment(String subjectKey, String experimentKey) {
-        return this.getStringAssignment(subjectKey, experimentKey, new SubjectAttributes());
+        return this.getStringAssignment(subjectKey, experimentKey, new EppoAttributes());
     }
 
     /**
-     * This function will return string assignment value
-     * 
-     * @param subjectKey
-     * @param experimentKey
-     * @param subjectAttributes
-     * @return
+     * Maps a subject to a variation for a given flag/experiment.
+     *
+     * @param subjectKey identifier of the experiment subject, for example a user ID.
+     * @param flagKey flagKey feature flag, bandit, or experiment identifier
+     * @return the variation string assigned to the subject, or null if an unrecoverable error was encountered.
```

Comment:
Our Javadocs were not fleshed out, so I did so for the `getStringAssigment()` functions. In a later PR, we can flesh out the rest of them if we want--I just didn't want to bloat this PR.